src/main/java/screens/login_registration/LogoutResponseFormatter.java

```diff
@@ -0,0 +1,17 @@
+package screens.login_registration;
+
+import use_cases.login_registration.logout_usecase.LogoutPresenter;
+import use_cases.login_registration.logout_usecase.LogoutResponseModel;
+
+public class LogoutResponseFormatter implements LogoutPresenter {
+
+    /**
+     * @param logout the logout response
+     * @return the successful logout view
+     */
+    @Override
+    public LogoutResponseModel prepareSuccessView(LogoutResponseModel logout) {
+        return logout;
+    }
+
+}
```

Comment:
It doesn't really matter in terms of functionality, but this should be the Presenter (and the presenter file in your use case should be the OutputBoundary